packages/opentelemetry-metrics/src/export/aggregators/Histogram.ts

```diff
@@ -48,13 +48,14 @@ export class HistogramAggregator implements HistogramAggregatorType {
     this._current.count += 1;
     this._current.sum += value;
 
+    this._lastUpdateTime = hrTime();
```

Comment:
Nit: move this up to be the first thing to update

Reply:
👍